### ISSUE TITLE
Need Itch Client for 64 bits Plz!

### ISSUE THREAD

**Issue by anonymous725195** [itchio/itch#2569]
I hate to put the games in the app data (.../User Folder), I wanted to put the games in the same folder as steam (Program Files x86) app data is a terrible place to put games, as it is hidden in the system and by the same organization

**Comment by ERmilburn02** [comment]
You know it asks where you want to install to, right?

**Comment by anonymous725195** [comment]
Yes but I am being required to give administrator permissions every time after installation

**Comment by leafo** [comment]
Closing out stale issues. Please try the latest version of the app and if you're still having a problem you can comment or create a new issue.

As far as I'm aware the Windows build is 64 bit application now.
